enterprise/server/test/integration/remote_bazel/remote_bazel_test.go

```diff
@@ -124,6 +125,7 @@ func clonePrivateTestRepo(t *testing.T) {
 	err = os.Chdir(repoDir)
 	require.NoError(t, err)
 	testshell.Run(t, repoDir, "git pull")
+	return repoDir
 }
 
 func resetFlags(t *testing.T) {
```

Comment:
can we get rid of `resetFlags` now that the flags are only being set on the subprocess?

Reply:
Yep good call - removed